tests/gbs/test_vibronic.py

```diff
@@ -41,7 +41,16 @@ class TestGBSParams:
     )
     d = np.array([0.2254, 0.1469, 1.5599, -0.3784, 0.4553, -0.3439, 0.0618])
 
-    U1, S, U2, alpha = vibronic.gbs_params(w, wp, Ud, d)
+    T = 300.0
+
+    t = np.array([1.19894657e-04, 6.07995705e-04, 1.25672831e-02, 3.34594410e-02, 4.15755027e-02,
+                  6.55350936e-02, 2.24605404e-01])
```

Comment:
Do we need this, since it's re-declared a few lines below?

Reply:
Will remove it.